django/db/backends/base/schema.py

```diff
@@ -942,16 +942,27 @@ def _get_index_tablespace_sql(self, model, fields, db_tablespace=None):
             return ' ' + self.connection.ops.tablespace_sql(db_tablespace)
         return ''
 
+    def _get_include_sql(self, model, include_fields):
+        if not include_fields:
+            return ''
+        include_columns = Columns(
+            model._meta.db_table,
+            [field.column for field in include_fields],
+            self.quote_name
+        )
+        return ' INCLUDE (%s)' % include_columns if include_columns else ''
```

Comment:
You'll want to return a `Statement` instead of interpolating here so aliases can be repointed if an `AddIndex` operation is included in a migration that also includes a rename or deletion operation and the index creation needs to be deferred.

```python
return Statement(
    ' INCLUDE (%(columns)s)',
    columns=columns
) if columns else ''
```